client/my-sites/domains/domain-management/transfer/transfer-page/index.tsx

```diff
@@ -58,12 +66,17 @@ const TransferPage = ( props: TransferPageProps ): JSX.Element => {
 		return <Breadcrumbs items={ items } mobileItem={ mobileItem } />;
 	};
 
-	return (
-		<Main className="transfer-page" wideLayout>
-			<BodySectionCssClass bodyClass={ [ 'edit__body-white' ] } />
-			{ renderBreadcrumbs() }
-			<Card>
+	const renderTransferOptions = () => {
+		const options = [];
+
+		if ( ! isDomainOnly ) {
+			const mainText = isMapping
+				? __( 'Transfer this domain connection to any administrator on this site' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **15** times:
`translate( 'Transfer this domain to any administrator on this site' )` *ES Score: 8*